src/Core/Resolvers/MsSqlQueryBuilder.cs

```diff
@@ -421,7 +421,7 @@ private string MakeOutputColumns(List<LabelledColumn> columns, string columnPref
         /// </summary>
         private string Build(LabelledColumn column, string columnPrefix)
         {
-            if (columnPrefix.IsNullOrEmpty())
+            if (string.IsNullOrEmpty(columnPrefix))
```

Comment:
this now matches correct usage of : https://learn.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=net-8.0

String doesn't have a built-in extension method `IsNullOrEmpty()`

might be worth doing search for `IsNullOrEmpty()` throughout codebase to ensure this isn't elsewhere and identify why intellisense/compilation process doesn't complain about this.

Reply:
looks like we had already fixed nearly all of these, just 1 more case which I added to this PR.

Reply:
@aaronburtle I am not sure if it is the case but I think I found one in `Core\Resolvers\SqlMutationEngine.cs` method `BuildAndExecuteInsertDbQueries():exposedColumnNames.IsNullOrEmpty()`

Reply:
@RubenCerna2079 , exposedColumnNames is **not** of String type.